docs/src/increase-observability/metrics.md

```diff
@@ -0,0 +1,251 @@
+---
+title: Monitor metrics
+weight: 5
+description: See all of the live infrastructure metrics available to give you an overview of resource usage.
+aliases:
+  - /dedicated/architecture/metrics.html
+---
+
+Platform.sh projects are accompanied by live infrastructure metrics that provide an overview of resource usage for environments.  
+
+Within the console, metrics can be found for an environment under **Metrics**:
+
+![Metrics dashboard](/images/metrics/all.png "0.65")
```

Comment:
Shouldn't this screenshot just show the navigation?
As per the [content style page](https://github.com/platformsh/platformsh-docs/blob/main/contributing/content-style.md#format-screenshots-for-sustainability).

Also we recommend:
> When adding images and other visual elements to the docs, make sure the same content is available to those who can't see the content. This includes people who don't see and people who have images off in their browser for whatever reason.
Use alternative text for images that add content to a page that isn't otherwise there. Imagine what you would write if you couldn't include the image.

But we don't add alt text, nor do we [provide examples](https://github.com/platformsh/platformsh-docs/blob/main/contributing/markup-format.md#images) on how to do it.

It seems the default hugo behavior was changed to provide as a second parameter the scaling instead of the alt text. I could not find it in the shortcodes and I didn't want to spend too much time on it if it's not a priority. Let me know what you think.